src/gui/window/settings_window.rs

```diff
@@ -146,8 +146,9 @@ pub fn draw_settings(
                 .spacing([40.0, 4.0])
                 .min_col_width(col_width)
                 .show(ui, |ui| {
-                    ui.label("Fullscreen: ");
-                    ui.checkbox(&mut state.fullscreen, "");
+                    if ui.button("Toggle Fullscreen").clicked() {
+                        state.fullscreen = true;
```

Comment:
Was this intentional btw? Did you mean `state.fullscreen = !state.fullscreen`? Also I'm not sure about removing the checkbox, IMO usually they're a bit better than toggles because they're more descriptive, I wonder what @MyBlackMIDIScore would think

Reply:
Yes, this was intentional. As soon as fullscreen is set, `state.fullscreen` is set to false